wye/workshops/views.py

```diff
@@ -36,11 +36,25 @@ def get_success_url(self):
         return super(WorkshopUpdate, self).get_success_url()
 
 
-class WorkshopToggleActive(views.LoginRequiredMixin,
-    views.CsrfExemptMixin, views.JSONResponseMixin,
-    generic.View):
+class WorkshopToggleActive(
+    views.LoginRequiredMixin, views.CsrfExemptMixin,
+    views.JSONResponseMixin, generic.UpdateView):
+
+    model = Workshop
+
+    def post(self, request, *args, **kwargs):
+        self.object = self.get_object()
+        response = self.object.toggle_active(request.user, **kwargs)
+        return self.render_json_response(response)
+
+
+class WorkshopAssignMe(
+    views.LoginRequiredMixin, views.CsrfExemptMixin,
+    views.JSONResponseMixin, generic.UpdateView):
+
     model = Workshop
 
     def post(self, request, *args, **kwargs):
-        response = self.model.toggle_active(**kwargs)
+        self.object = self.get_object()
+        response = self.object.assign_me(request.user, **kwargs)
```

Comment:
yes, it seems I was missing this file wye/profiles/migrations/0004_auto_20151017_1321.py

Updated my code